pyproject.toml

```diff
@@ -19,6 +19,8 @@ dependencies = [
     "frictionless[json,parquet]>=5.6.3",
     "tomli>=2.0.1",
     "genson>=1.2.2",
+    "thrift>=0.16.0",
```

Comment:
Feel like this should be in a `hive-metastore` group.

Reply:
right! I'll clean up the groups and I'll continue working on adding the rest of the functionality. This was a very quick PR just to share the generated client with you.